app/routes/account.$accountId._index/view.tsx

```diff
@@ -1,25 +1,53 @@
-import { Badge, Card, Group, SimpleGrid, Stack, Text, Title } from "@mantine/core"
-import { useNavigation } from "@remix-run/react"
+import { Card, SimpleGrid, Skeleton, Stack, Text, Title } from "@mantine/core"
+import { useLocation, useNavigation } from "@remix-run/react"
 import React from "react"
+import InsightsControls from "app/components/InsightsControls"
 import TitledCard from "~/components/TitledCard"
-import { AnalyticsRelaysAggregated } from "~/models/dwh/sdk/models/AnalyticsRelaysAggregated"
-import { AnalyticsRelaysTotal } from "~/models/dwh/sdk/models/AnalyticsRelaysTotal"
+import { Account, Blockchain, PortalApp } from "~/models/portal/sdk"
 import { AccountAppsOverview } from "~/routes/account.$accountId._index/components/AccountAppsOverview"
-import { ChartPeriodSelector } from "~/routes/account.$accountId._index/components/ChartPeriodSelector"
 import { OverviewSparkline } from "~/routes/account.$accountId._index/components/OverviewSparkline"
 import useAggregateChartData from "~/routes/account.$accountId._index/hooks/useAggregateChartData"
+import { AccountInsightsData } from "~/routes/account.$accountId._index/route"
 import { getTotalErrors } from "~/utils/chartUtils"
 import { commify } from "~/utils/formattingUtils"
 
-type AccountInsightsViewProps = {
-  total: AnalyticsRelaysTotal
-  aggregate: AnalyticsRelaysAggregated[]
+type AccountInsightsViewProps = Pick<AccountInsightsData, "total" | "aggregate"> & {
```

Comment:
this prop looks like it could just be AccountInsightsData type. Why do this?

Reply:
Good catch, fixed it 👌